src/main/java/com/ft/methodestorypackagemapper/model/StoryPackage.java

```diff
@@ -34,6 +35,7 @@ public String getPublishReference() {
         return publishReference;
     }
 
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'", timezone = "UTC")

```

Comment:
can a value like this be found in DateTimeFormatter? Is `DateTimeFormatter.ISO_INSTANT` the same? (if you worry on the milliseconds, i think they are resolved dynamically). If yes, please change, if not, never mind.